_includes/markdown/Typescript.md

```diff
@@ -0,0 +1,188 @@
+The purpose of this document is to help guide Engineers in the usage of TypeScript at 10up projects. The goal here is not to teach TypeScript but rather explain the 10up stance on when to use TypeScript alongside general recommendations.
+
+## Why TypeScript
```

Comment:
It might be worth mentioning that many industry standard tools and packages (Sanity, NextJS, Storybook etc) have moved to TypeScript as well - its not something we're considering in isolation, its a noticeable industry pattern that's shifting development for the better

Reply:
Perhaps there should be an example of importing a type from one of these packages and using it in some example code. I think we should try to encourage engineers to import and use types from packages where possible rather than write their own (unless it will slow them down to do so)